app/helpers/summaries_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module SummariesHelper
+  def date_for_summary_view(date)
+    date.strftime("%-d %B %Y")
+  end
```

Comment:
Might want to use this is the confirmation components where we playback the dates